fbw-a380x/src/wasm/systems/a380_systems/src/hydraulic/autobrakes.rs

```diff
@@ -209,12 +223,15 @@ impl A380AutobrakeController {
     const TARGET_TO_SHOW_DECEL_IN_RTO_MS2: f64 = -2.7;
     const TARGET_TO_REMOVE_DECEL_IN_RTO_MS2: f64 = -2.;
 
+    const KNOB_SOLENOID_DISARM_DELAY: Duration = Duration::from_millis(1000);
+
     pub fn new(context: &mut InitContext) -> A380AutobrakeController {
         A380AutobrakeController {
             armed_mode_id: context.get_identifier("AUTOBRAKES_ARMED_MODE".to_owned()),
             decel_light_id: context.get_identifier("AUTOBRAKES_DECEL_LIGHT".to_owned()),
             active_id: context.get_identifier("AUTOBRAKES_ACTIVE".to_owned()),
             rto_mode_armed_id: context.get_identifier("AUTOBRAKES_RTO_ARMED".to_owned()),
+            ground_speed_id: context.get_identifier("GPS GROUND SPEED".to_owned()),
```

Comment:
I assume this comes from the GPS in the real airplane (which is not implemented yet)?